### PR TITLE
ADD Plugin obsidian-markdown-file-suffix

### PR DESCRIPTION
**Pull request by git-no**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/swissmation/obsidian-markdown-file-suffix

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Review by git-no — COMMENTED** [review]
ADD plugin to json file

**Comment by git-no** [comment]
Seems the workflow has an error. Can we run the workflow again? I do not understand the error message.

`Run actions/github-script@v6
[101](https://github.com/obsidianmd/obsidian-releases/runs/7837117627?check_suite_focus=true#step:4:102)
TypeError: Cannot read properties of undefined (reading 'split')`

**Comment by liamcain** [comment]
- [name: Upload styles.css](https://github.com/git-no/obsidian-markdown-file-suffix/blob/dd2fbcf00f1ac57886875794da41c7120a7fb062/.github/workflows/release.yml#L78) You don't need to upload the css file since it's not used.

**Comment by liamcain** [comment]
@git-no, can you rebase this PR off of the latest master when you have a chance? Then I think this can go in.

File: community-plugins.json [hunk 1]
```diff
@@ -4668,5 +4668,13 @@
         "description": "Create folder of snippets to activate them in one click !",
         "repo": "Mara-Li/obsidian-group-snippets",
         "branch": "master"
+    },
+    {
+        "id": "obsidian-markdown-file-suffix",
```
Comment (liamcain) [inline community-plugins.json:4673]: I suggest removing the `obsidian-` prefix. It's not needed. Also the name and ID should probably better match the use of the plugin. Maybe just `mdx-svx-extension-support`.

File: community-plugins.json [hunk 2]
```diff
@@ -4668,5 +4668,13 @@
...
+        "name": "More Markdown file suffix (.mdx/.svx).",
```
Comment (liamcain) [inline community-plugins.json:4674]:
```suggestion
        "name": "MDX/SVX file support",
```

Maybe something like this is better?

**Comment by liamcain** [comment]
Hi @git-no, I think this can be merged, but you will need to manually update this pull request to move your plugin to the bottom of the plugin list. I don't have permission for your fork of the repo.

**Comment by ericaxu** [comment]
Added through this commit: https://github.com/obsidianmd/obsidian-releases/commit/cfb863cb1d27ba83e31b88e9b71f3356d56d3992

Please note that I removed the ending period in the name, since it's inconsistent with other plugin names. Please open a new PR if you still want it, thanks! @git-no

**Pull request closed without merging**